modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc

```diff
@@ -5,15 +5,20 @@
 [id="virt-enabling-mac-address-pool-for-namespace-cli_{context}"]
 = Enabling a MAC address pool for a namespace in the CLI
```

Comment:
Would it make sense to change the title to `Re-enabling a MAC address pool for a namespace in the CLI`?